docs/specs/oidc.md

```diff
@@ -300,24 +317,25 @@ To indicate the authenticator used in authentication, `amr` claim is used in OID
 - If password authenticator is used: `pwd` is included.
 - If any OTP (TOTP/OOB-OTP) is used: `otp` is included.
 - If WebAuthn is used: `hwk` is included.
+- If Biometric is used: `x_biometric` is included.
```

Comment:
I think we should also include as much detailed information in the amr, if it is a list allow multiple values? Eg if it is iOS FaceID, than ['face', 'x_biometric']. If not than just the x_biometric is fine.

Reply:
Agree. Opened this issue to follow up https://github.com/authgear/authgear-server/issues/1295